src/Microsoft.ML.Onnx/SaveOnnxCommand.cs

```diff
@@ -113,9 +114,10 @@ public override void Run()
             }
         }
 
-        private void GetPipe(OnnxContextImpl ctx, IChannel ch, IDataView end, out IDataView source, out IDataView trueEnd, out LinkedList<ITransformCanSaveOnnx> transforms)
+        internal static void GetPipe(OnnxContextImpl ctx, IChannel ch, IDataView end, out IDataView source, out IDataView trueEnd, out LinkedList<ITransformCanSaveOnnx> transforms)
         {
-            Host.AssertValue(end);
+            Contracts.AssertValue(end);
```

Comment:
>Contracts [](start = 12, length = 9)

Why not `ch`?

Reply:
No problem.

---
In reply to: [245428925](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245428925) [](ancestors = 245428925)